scalafix-tests/output/src/main/scala-3/test/organizeImports/DeduplicateImportees.scala

```diff
@@ -0,0 +1,7 @@
+package test.organizeImports
+
+import scala.collection.immutable.Vector
+import scala.collection.immutable.{Map => Dict}
+import scala.collection.immutable.{Set as _, *}
```

Comment:
The first line is [preserved](https://github.com/liancheng/scalafix-organize-imports/issues/127) while the second is generated by Scalafix, thus the inconsistency. We need to fix that - just not sure if it has to be in this PR as it does not look trivial.

Reply:
Actually, preserving original syntax is also what is causing a few tests to fail as:
- the sorting is done on pretty-printed importers even though the original syntax remains
- `foo.{bar => qux}` is rewritten `foo.bar as qux` with the Scala 3 dialect

As a result, the order is changed for no apparent reason: a preserved `foo.{bar => qux}` no longer follows `import foo.wiz` if the source is Scala 3.